meow/scheduler/models.py

```diff
@@ -297,3 +297,12 @@ class EmailNotification(models.Model):
 
     def __str__(self):
         return self.name + " <" + self.email_address + ">"
+
+class PostHistory(models.Model):
+    smpost = models.ForeignKey(
+            SMPost, blank=True, null=True, related_name='+', on_delete=models.SET_NULL)
+    post_facebook = models.TextField(null=True, blank=True)
+    post_instagram = models.TextField(null=True, blank=True, default="")
+    last_edit_user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, blank=True, null=True, related_name='+', on_delete=models.SET_NULL)
```

Comment:
Looks good! Just a few things on the ForeignKeys.
Since every post has a `last_edit_user`, I think we should remove the `blank=True, null=True`. Same idea for the smpost. Then because those fields can't be set null, `on_delete` should equal `models.CASCADE` instead.
Lastly, I don't think we need the related name for `smpost` and `last_edit_user`